web/src/store/index.js

```diff
@@ -335,6 +365,9 @@ const actions = {
             case plot_types.ANALYSIS:
               ({ data: d } = await CSVService.getAnalysis(dataset_id, name, args));
               break;
+            case plot_types.CLIENT_ONLY:
```

Comment:
I looked at `BoxPlotTile.vue` and I didn't see a compelling reason to use the `plotData` mixin for this plot.  You don't appear to be using any of the features of that mixin except for `this.dataset`.  Rather than add this `CLIENT_ONLY` type, would it make sense to just map `dataset`  with `mapGetters` and drop the mixin from that tile altogether?

I'd prefer to keep the plot state stuff for situations where plots are being generated server-side.